server/package.json

```diff
@@ -5,9 +5,10 @@
   "author": "",
   "license": "ISC",
   "scripts": {
-    "db:seed": "node db/scripts/seed.js",
-    "watch": "NODE_ENV=development nodemon main.js",
-    "start": "NODE_ENV=production node main.js",
+    "db:seed": "tsc db/scripts/seed.ts",
+    "watch": "NODE_ENV=development nodemon main.ts",
+    "start": "NODE_ENV=production npm run build && node dist/main.js",
```

Comment:
Don't we need to change lambda entrypoints? I don't know where they are.

Reply:
yeah, I am working on that but I am trying to figure out how to resolve the issue with `custom_modules` first.